doc/classes/@GDScript.xml

```diff
@@ -828,7 +828,7 @@
 			<return type="float">
 			</return>
 			<description>
-				Returns a random floating point value between 0 and 1.
+				Returns a random floating point value on the interval [0, 1].
```

Comment:
(guess it is a bit late, but..) Is it `in` or `on` the interval? The other two places use `in`.